envMDT.sh

```diff
@@ -1,5 +1,40 @@
 #!/bin/bash
 
+drop_from_path()
+{
+   # Assert that we got enough arguments
+   if test $# -ne 2 ; then
+      echo "drop_from_path: needs 2 arguments"
+      return 1
+   fi
+
+   local p=$1
+   local drop=$2
+
+   newpath=`echo $p | sed -e "s;:${drop}:;:;g" \
+                          -e "s;:${drop}\$;;g"   \
+                          -e "s;^${drop}:;;g"   \
+                          -e "s;^${drop}\$;;g"`
+}
+
+### main ###
+
+## clean old environment
+if [ -n "${MDTROOT}" ] ; then
+	old_mdtroot=${MDTROOT}
+
+   	if [ -n "${LD_LIBRARY_PATH}" ]; then
+		drop_from_path "$LD_LIBRARY_PATH" "${old_mdtroot}/cpp"
+		LD_LIBRARY_PATH=$newpath
+
+		drop_from_path "$LD_LIBRARY_PATH" "${WCSIMDIR}"
+		LD_LIBRARY_PATH=$newpath
+
+		drop_from_path "$LD_LIBRARY_PATH" "${WCRDROOT}"
+		LD_LIBRARY_PATH=$newpath
+	fi
+fi
+
 export MDTROOT=`pwd`
```

Comment:
consider modifying procedure to be able to run this script from anywhere

Reply:
done